.github/workflows/ci.yml

```
@@ -71,7 +71,7 @@ jobs:
       shell: bash -l {0}
       run: |
         # This will not install double precision, needs to be replaced with a fresh build
-        conda install gromacs -c bioconda -c conda-forge -y
+        conda install gromacs=4.6.5 -c bioconda -c conda-forge -y
```

What about trying 2019 or 2018? Versions 2019.1 and 2018.6 were previously being pulled down, i.e. these CI logs from a week ago: https://github.com/leeping/forcebalance/runs/2115435511

```suggestion
        conda install gromacs=2019.1 -c bioconda -c conda-forge -y
```